abeja/tracking/__init__.py

```diff
@@ -11,7 +13,31 @@
 
 
 class Tracking:
-    def __init__(self, total_steps: Optional[int] = None):
+    """Model Tracking
+
+    Usage:
+        .. code-block:: python
+
+            from abeja.tracking import Tracking
+
+
+            NUM_EPOCHS = 10
+            for epoch in range(NUM_EPOCHS):
+                with Tracking(total_steps=NUM_EPOCHS) as tk:
+                    # Write your modeling code here.
+                    tk.log_description("Example")
+                    tk.log_param(key="LR", value="0.03")
+                    tk.log_param(key="DROPOUT", value="0.7")
+                    tk.log_metric(key="main/acc", value=0.95)
+                    tk.log_metric(key="main/loss", value=0.01)
+                    tk.log_artifact(filepath='filepath_to_your_model', delete_flag=True)
+    """
+
+    def __init__(
+            self,
+            total_steps: Optional[int] = None,
+            logger: Optional[Logger] = logging.logger):
```

Comment:
For testability, fix to inject logger.
It was not possible to pass tests except for this way, because this module is callable ( users can use this like `import abeja.tracking; abeja.tracking()` ) and which prevent from patching with  `@patch('abeja.tracking.logger')`.